package.json

```diff
@@ -75,13 +76,14 @@
     "test": "cross-env NODE_ENV=test node scripts/test.js --env=./scripts/custom-test-env.js --watchAll --coverage",
     "eject": "react-scripts eject",
     "lint:check": "eslint \"**/*.{ts,tsx}\" --max-warnings=0",
-    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix",
-    "format:fix": "prettier --write \"**/*.{ts,tsx,json,scss,css}\"",
+    "lint:fix": "eslint --fix",
+    "format:fix": "prettier --write",
```

Comment:
No not make changes to these scripts as they are used in the workflows for holistic checking. In `.lintstagedrc.json` run the `prettier` and `eslint` commands directly insteading of creating a new script for them.

Reply:
Because lint-staged provides the staged files based on glob patterns specified in .lintstagedrc.json to these commands, So the glob patterns hard-coded in the commands need to be removed.

Image from lint-staged npm page:-
![Screenshot from 2023-12-30 19-18-04](https://github.com/PalisadoesFoundation/talawa-admin/assets/121512095/51dc4b51-e8f8-457e-b8d2-b88a2ae8e448)

Reply:
I can make a video if you want but .yml files are not programmed to be added for staged files. They were not programmed so earlier as well for lint check.

Reply:
I get why you removed the globs from the scripts, but we can't do that as these scripts are required as in the workflows we used. 

You can call the eslint and prettier commands directly throught lintstagedrc. 

The video is optional, though it gives the reviewer the confidence existing workflows aren't breaking. The yml check was necessary as we had previous PRs where the same was not working and caused all the PR workflows to fail.

Reply:
Video as requested:-

https://github.com/PalisadoesFoundation/talawa-admin/assets/121512095/9f9634a7-e195-4c49-93b1-a7d94a9be210

As shown in video, only staged files are checked. If I make the changes requested then all files will be checked for linting by default. A possible work around can be to make new npm scripts for lint-staged and leave the old ones as they are.

Shall I do this way?